File: nj2k/testData/newJ2k/switch/nestedSwitch.java
//method
public String foo(int i, int j) {
    switch (i) {
        case 0:
            switch (j) {
                case 1:
                    return "0, 1";
                default:
                    return "0, x";
            }
        case 1:
            return "1, x";
        default:
            return "x, x";
    }
}

File: kotlin-ide/j2k/old/testData/fileOrElement/switch/nestedSwitch.kt
fun foo(i: Int, j: Int): String {
    when (i) {
        0 -> when (j) {
            1 -> return "0, 1"
            else -> return "0, x"
        }
        1 -> return "1, x"
        else -> return "x, x"
    }
}